shared/headers/create-address.yml

```diff
@@ -0,0 +1,7 @@
+send_date:
+  description:
+    A timestamp of the date on which a mailpiece will be dispatched
+    for production. Beyond this date, the mailpiece can't be cancelled.
+  schema:
+    type: string
+    example: "2017-09-05T17:47:53.967Z"
```

Comment:
This file is missing a final carriage return. You can configure VS Code to clean your files for you. In Settings => Text Editor => Files, enable Insert File Newline, Trim Final Newlines and Trim Trailing Whitespace and your files will always be lovely. :slightly_smiling_face: